app/assets/javascripts/components/eventNoteType.js

```diff
@@ -0,0 +1,22 @@
+const textMap =  {
+  300: 'SST Meeting',
+  301: 'MTSS Meeting',
+  302: 'Parent conversation',
+  304: 'Something else',
+  305: '9th Grade Experience',
+  306: '10th Grade Experience'
+};
+export function eventNoteTypeText(eventNoteTypeId) {
+  return textMap[eventNoteTypeId] || 'Other';
+}
+
+const colorMap =  {
+  300: '#31AB39',
+  301: '#EB4B26',
+  302: '#CDD71A',
+  305: '#139DEA',
+  306: '#333333'
+};
```

Comment:
Similarly, I think these colors are experimental at this point, and all subject to change next week.